All Core Devs Meetings/meeting 94.md

```diff
@@ -0,0 +1,437 @@
+  
+**Danny** - Do you know the time and estimated time crossing the like, worst case, just like right now?
+  
+**Vitalik** - My understanding is that it's something like 30 seconds in my my proposed numbers would push it down to about 10.
+  
+**Alex Vlasov** - I don't know if someone has a statistics on how many calls on average, does one transaction make a I assume, taking into account a limited contract size and current approach to modularity with a lot of delegates calls, would it potentially impose a burden on a modern style of making smart contracts that will make them more expensive, especially something fancy, one like dex aggregators or just hexes themselves? So there is some statistics like this. 
+  
+**Vitalik** - Yeah, so. We actually do have a way of estimating this indirectly in the way that we do that is that we've we we've done numbers on estimating witnesses for the purpose of kind of one point X stuff. And so we've get we've gotten witnesses. So I think is somewhere around six hundred, six hundred kilobytes or so. And so from those numbers, we know that the number of these calls is going to be such that. I think as I as I recall, it would be somewhere, somewhere in the mind of 10 to 20 percent, 10 and 20 percent range in terms of how much more expensive average the average things would get. But then we could definitely estimate this more precisely.
+  
+**Alex Vlasov** - kind of related question. But it's more from a little perspective with the. Looking at the location on the desk would still it would have like highlights defined dnd start with it, but then you would get a huge batch of data potentially for free. Would it be reasonable to maybe increase the contract limit a little to kind of compensate that?
+  
+**Vitalik** - I mean, I'd definitely be happy to like I would personally definitely be OK with that and then well, the thing that we want to keep in mind also in the long run is that in the long run, we would probably wants to move toward a regime where contract contract code has either a much larger limit or no limit because we have code virtualization and we will be charging for a chunk. So and and in that context, like basically having your code be in one contract versus having your code being split among a bunch of a bunch of different addresses would be it would be almost equivalent potentially. But I would definitely see in terms a personal with a slight increase in the max contract size, especially if it helps alleviate the effects to developers.
+  
+**Hudson** - Ok, Barton, did you have something? I think you're muted, Martin, if you're talking.
```

Comment:
Ok, Martin, did you have something? I think you're muted, Martin, if you're talking.